src/components/drawer/drawer.stories.tsx

```diff
@@ -467,7 +467,7 @@ export const CustomSidebar: ComponentStory<typeof Drawer> = (args) => {
     setCurrentPage(newPage);
   };
   return (
-    <div>
+    <Box>
```

Comment:
This can be investigated in a separate ticket as this work is needed asap

Reply:
sure, it's not important or anything - but since it seemed we were replacing `div` with `Box` as part of the PR even though that's surely separate from the focus styles, I thought it worth commenting on. Definitely non-blocking as I already said.